src/ott/geometry/distrib_cost.py

```diff
@@ -0,0 +1,82 @@
+# Copyright OTT-JAX
```

Comment:
This file needs to be added to `__init__` in `src/ott/geometry`.

Reply:
thanks!